### PR TITLE
patch for add user to implement email password temp login flow

### PR DESCRIPTION
**Pull request by AbrahamGeorge8547**

### CODE REVIEW

File: db/migration/models.sql [hunk 1]
```diff
@@ -9,8 +9,10 @@ CREATE TABLE users (
     updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
     username VARCHAR(255) UNIQUE NOT NULL,
     name VARCHAR(255) NOT NULL, 
-    public_key TEXT NOT NULL,
-    ecc_pub_key TEXT NOT NULL
+    public_key TEXT,
```
Comment (akash1729) [inline db/migration/models.sql:12]: use the type of key as prefix for public_key similar to ecc_pub_key ?

File: dtos/user_dto.go [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package dto
 
 type CreateUser struct {
-	UserName  string `json:"username"`
-	Name      string `json:"name"`
-	PublicKey string `json:"publicKey"`
+	UserName     string `json:"username"`
+	Name         string `json:"name"`
```
Comment (akash1729) [inline dtos/user_dto.go:5]: Not sure we need a Name to create a user. We can let the invited user fill in his name

**Comment by akash1729** [comment]
LGTM. Minor comments

**Review by akash1729 — APPROVED** [review]

File: dtos/user_dto.go (hunk 2 above)
Comment (AbrahamGeorge8547) [reply dtos/user_dto.go:5]: possible hack where the user changes the name to confuse other users. intern changing name to cto

**Pull request merged**